polygeist/lib/polygeist/Passes/ConvertToLLVMABI.cpp

```diff
@@ -0,0 +1,123 @@
+//===- ConvertToLLVMABI.cpp - Convert to LLVM Pointer ABI -----------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This pass currently only handles SYCL Kernel functions.
```

Comment:
Ok, this is temporary, correct? We will need this pass to work for any external function (device or host).

Reply:
Yes, only handling `gpu.func` with kernel attribute is a short-term limitation in this PR.